### ISSUE TITLE
Uncaught TypeError: Cannot read property 'score' of undefined

### ISSUE THREAD

**Issue by swiftanthony** [cozmo/jsQR#44]
I am using jsQR with Quagga to scan qrcode using web camera. Getting this error:

```
Uncaught TypeError: Cannot read property 'score' of undefined
```

Happening at this line - https://github.com/cozmo/jsQR/blob/master/dist/jsQR.js#L9911

**Comment by cozmo** [comment]
I believe this is the same issue that was (I thought?) fixed in https://github.com/cozmo/jsQR/issues/41 - Can you confirm you're using the latest version `v1.0.1`?

**Comment by swiftanthony** [comment]
True, it is fixed in `1.0.1`, I was still on `1.0.0`

Thank you.

**Comment by Ridder90** [comment]
Hi Swiftanthony,
I am very curious to your implementation on jsQR with Quagga. I am not alone.
Do you mind to share your insights? Thanks!

https://github.com/serratus/quaggaJS/issues/317
